### ISSUE TITLE
Sensor not updating

### ISSUE THREAD

**Issue by itm1960** [arsaboo/homeassistant-config#8]
Hi
This component looks like exactly what I am looking for, but I am having problems getting my life360 sensor to update.
I have copied life360.py to my custom_components/sensor folder, and have the following entries in my configuration.yaml:
```
mqtt:
  embedded:
    listeners:
      default:
        max-connections: 5000
        bind: 192.168.1.36:1883
        type: tcp
      ws-1:
          bind: 192.168.1.36:8087
          type: ws
    auth:
      plugins: ['auth.anonymous']
      allow-anonymous: true
  broker: 192.168.1.36
  port: 1883
  
sensor:
  - platform: life360
    name: life360_ian
    username: "mylife360user@gmail.com"
    password: "mylife360password"
    mqtt_topic: "life360/ian"

device_tracker:
  - platform: owntracks
```
I can see the life360 sensor on my HA console, but it never gets updated. I'm running HA v0.65.0 on Lubuntu 16.10. MQTT seems to be running OK.
The only part of the instructions in your code that I didn't understand was the part that said I needed to copy the life360 package from your repository - I don't know what this means?(!)

Have you any idea why my life360 sensor is not getting updated?

**Comment by arsaboo** [comment]
It should work. Wondering if the issue is with the embedded broker (that is the only thing that is different between our configurations).

**Comment by itm1960** [comment]
I've installed MQTT.fx and it is showing that the following topics are being updated every minute or so on my HA server:
```
life360/ian
owntracks/my name/my name
owntracks/my wife’s name/my wife’s name
```
...so it looks like MQTT is working OK, But I notice this error message whenever I restart HA:
`Unable to load /config/known_devices.yaml: Config file not found: /config/known_devices.yaml`
I'm guessing that it's the known_devices.yaml that is used to update the sensor status on the HA console?
I have no idea why this file isn't being created as it should.

**Comment by arsaboo** [comment]
You can delete the file and restart HA. it will be recreated.

**Comment by itm1960** [comment]
The known_devices.yaml file never actually gets created, so there's nothing to delete.
As an experiment, I created a different device tracker using the ASUSWRT platform (i.e. using my Asus router to detect presence). This did create a known_devices.yaml file, containing all of the devices connected to my Asus router. It didn't contain any owntracks entries though. This at least demonstrated that HA has the necessary privileges to create the known_devices.yaml file. It just never gets created by the owntracks platform.
???

**Comment by arsaboo** [comment]
Owntracks should also create an entry in known_devices.yaml. If the topic is owntracks/alok/phone, the entry in known_device.yaml should be `alok_phone:`. If you don't see that, there is something wrong with Owntracks configuration.

**Comment by itm1960** [comment]
If I subscribe to the MQTT topics on the HA server I can see:
```
life360/ian
owntracks/ian/ian
owntracks/gill/gill
```
These are all being updated every minute with correct location and battery data etc, which suggests that owntracks is configured OK? The problem is that the HA device tracker that isn't being created.
The only owntracks config that I have is:
```
device_tracker:
  - platform: owntracks
```
...as the owntracks tracker is being populated by the life360.py code instead of an owntracks smartphone app.

**Comment by arsaboo** [comment]
you should see `ian_ian` and `gill_gill` entries in your device tracker. Configure the basic OT first before you start working on Life360. I think that is where your problem lies.

**Comment by itm1960** [comment]
OK I removed the Life360 sensor entry from my config, leaving only the owntracks device_tracker section (as above). Is this what you mean by configuring the basic OT? Or is there more that I need to do?

Anyway I then restarted HA. I got this error again:
```
Unable to load /config/known_devices.yaml: Config file not found: /config/known_devices.yaml
```
MQTT.fx now shows no MQTT topics being published on my HA server at all, and there are no device trackers in my HA console. Not sure where to go from here???
